samples/custom_module/dynamic/test/example.mlir

```diff
@@ -41,4 +52,13 @@ module @example {
 
     return
   }
+
+  // CERROR-LABEL: EXEC @error
+  func.func @error() {
+    // Show an example of dumping a custom error message.
+    // CERROR-NEXT: UNKNOWN; Forced failure; while invoking C++ function custom.error;
```

Comment:
you just want `UNKNOWN` - strings are optional

Reply:
darn